src/three-components/cube_uv_reflection_fragment.glsl.js

```diff
@@ -0,0 +1,97 @@
+export default /* glsl */ `
+#ifdef ENVMAP_TYPE_CUBE_UV
+
+#define cubeUV_faceSize (256.0)
+#define cubeUV_maxMipLevel (8.0)
+
+int getFace(vec3 direction) {
```

Comment:
Looks like we have another share-able utility here. Maybe we can group all of these in a common module...